monitoring/monitoring-overview.adoc

```diff
@@ -0,0 +1,34 @@
+:_content-type: ASSEMBLY
+[id="monitoring-overview"]
+= Monitoring overview
+include::modules/common-attributes.adoc[]
+:context: monitoring-overview
+
+toc::[]
+
+[id="about-openshift-container-platform-monitoring"]
+== About {product-title} Container Platform monitoring
+
+{product-title} includes a pre-configured, pre-installed, and self-updating monitoring stack that provides *monitoring for core platform components*. A cluster administrator can xref:../monitoring/configuring-the-monitoring-stack.adoc#configuring-the-monitoring-stack[configure the monitoring stack] with the supported configurations. You also have the option to enable xref:../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[monitoring for user-defined projects]. {product-title} delivers monitoring best practices out of the box. A set of alerts are included by default that immediately notify cluster administrators about issues with a cluster. Default dashboards in the {product-title} web console include visual representations of cluster metrics to help you to quickly understand the state of your cluster.
+The {product-title} web console provides xref:../monitoring/accessing-third-party-uis.adoc#accessing-third-party-uis[third-party integrated UIs] such as metrics, alerts, and review of monitoring dashboards. You can access these third-party user interfaces (UIs) either through the web console or the CLI. You can use the **Administrator** or ** Developer** perspective in the {product-title} monitoring dashboard to perform the following tasks:
+
+* xref:../monitoring/managing-metrics.adoc#managing-metrics[Manage metrics]
```

Comment:
You could potentially integrate this cross references into the lines that introduce metrics, alerts and dashboards. This would be consistent with the other links in the section to the assemblies in the Monitoring book, which are not in a bulleted format.

Reply:
I was in two minds, whether to create a bullet list so it's structured or to keep consistency with existing content. I will go with your suggestion